presidio-anonymizer/setup.cfg

```diff
@@ -0,0 +1,7 @@
+[bdist_wheel]
+# This flag says to generate wheels that support both Python 2 and Python
+# 3. If your code will not run unchanged on both Python 2 and 3, you will
+# need to generate separate wheels for each Python version that you
+# support.
+universal=1
```

Comment:
This means that we can run on Python 2 which isn't the case.

Reply:
I thought I saw this in the analyzer as well so I assumed it's ok.

Reply:
I removed it.

Reply:
You right! I'll remove it from there as well.